corehq/apps/app_manager/views/modules.py

```diff
@@ -207,7 +207,7 @@ def get_mod_dict(mod):
         }
 
     return {
-        'modules': [get_mod_dict(m) for m in app.modules if m.module_type == 'basic'],
+        'modules': [get_mod_dict(m) for m in app.modules if m.module_type != 'shadow'],
```

Comment:
good point, shadow modules don't work with reporting modules. not sure about care plan, but don't think it's worth the time/effort to investigate. fixed.